.circleci/workflows.yml

```diff
@@ -83,28 +90,29 @@ jobs:
           at: /
 
       - restore_cache:
-          key: python-3.10.6-v1
+          key: python-3.10.6-v2
 
       - run: pyenv install --skip-existing 3.10.6
       - run: pyenv global 3.10.6
-      - run: pip install -U pip poetry
+      - run: pip install -U pip
+      - run: pip install poetry==1.2.2
 
       - restore_cache:
           keys:
-            - beamer-dependencies-v4-{{ checksum "poetry.lock" }}
-            - beamer-dependencies-v4
-      - run: poetry install
-      - run: echo "source $(poetry env info -p)/bin/activate" >> ~/.bashrc
+            - beamer-dependencies-v5-{{ checksum "poetry.lock" }}
+            - beamer-dependencies-v5
+      - run: poetry install -vvv
```

Comment:
Do we still need the verbose logging (-vvv) here?

Reply:
We don't strictly need it now, but I left it in case something goes wrong in the future with poetry, we have the whole log right away, without having to rerun the workflow again.

But if you think we should just remove it, I will remove it.

Reply:
Nah, just leave it. We can remove it in the futrure.